### PR TITLE
Feat/info cards

### PR DESCRIPTION
**Pull request by TheLeoB**
## Description (optional)
KYC logic is implemented, more needs to be added once savings API is hooked up and we can detect if balances, interest, etc. 

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/scenes/Interest/InterestInfo/index.tsx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+                <Button
+                  nature='primary'
+                  data-e2e='verifyIdentityBorrow'
+                  style={{ marginTop: '20px' }}
+                  disabled={val.kycState !== 'NONE'}
+                  onClick={() =>
+                    this.props.identityVerificationActions.verifyIdentity(2)
+                  }
+                >
+                  {val.kycState === ('UNDER_REVIEW' || 'PENDING') ? (
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Interest/InterestInfo/index.tsx:62]: This syntax won't actually work the way you want it to.

The reason for this is that `('UNDER_REVIEW' || 'PENDING')` _always_ evaluates to `UNDER_REVIEW`

Comment (plondon) [reply packages/blockchain-wallet-v4-frontend/src/scenes/Interest/InterestInfo/index.tsx:62]:
```
kycState = 'PENDING'
=> "PENDING"
kycState === ('UNDER_REVIEW' || 'PENDING')
=> false
```

File: packages/blockchain-wallet-v4-frontend/src/scenes/Interest/InterestSummary/index.tsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+const AmountColumn = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+`
+const Separator = styled.div`
+  border: solid 1px ${props => props.theme.grey000};
+`
+
+class InterestSummary extends PureComponent<Props & State> {
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Interest/InterestSummary/index.tsx:42]: Should be:

PureComponent<Props, State>

File: packages/blockchain-wallet-v4-frontend/src/components/Layout/index.ts [hunk 3]
```diff
@@ -69,6 +69,10 @@ export const SceneSubHeaderText = styled(Text)`
   color: ${props => props.theme.grey600};
   font-weight: 500;
   margin-bottom: 30px;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/components/Layout/index.ts:75]: This is a pretty global component, I have no problem with the change but have you checked all the spots look good?

File: packages/blockchain-wallet-v4-frontend/src/scenes/Borrow/BorrowPax/index.tsx [hunk 4]
```diff
@@ -50,7 +50,7 @@ class BorrowPax extends PureComponent<Props & State> {
                   this.props.identityVerificationActions.verifyIdentity(2)
                 }
               >
-                {val.kycState === 'UNDER_REVIEW' ? (
+                {val.kycState === ('UNDER_REVIEW' || 'PENDING') ? (
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Borrow/BorrowPax/index.tsx:53]: See comment on InterestInfo/index.tsx

**Comment by plondon** [comment]
I resolved some merge conflicts, can you check all looks good?
```
CONFLICT (content): Merge conflict in packages/blockchain-wallet-v4-frontend/src/scenes/Interest/index.tsx
```

File: packages/blockchain-wallet-v4-frontend/src/components/Layout/index.ts (hunk 3 above)
Comment (TheLeoB) [reply packages/blockchain-wallet-v4-frontend/src/components/Layout/index.ts:75]: It's the subheader underneath empty states. Earn Interest is the only one with a 'Learn More' link that needed to be centered to the Text, so it didn't affect the others.

**Comment by TheLeoB** [comment]
Changes/merge conflicts resolved. Thanks Phil.

**Pull request merged**